hack/test.sh

```diff
@@ -21,7 +21,8 @@ set -o pipefail
 
 export CGO_ENABLED=0
 
-TARGETS=($(go list ./pkg/... | grep -v "github.com/vmware-tanzu/velero/pkg/builder"))
+TARGETS=($(go list ./pkg/... | grep -v "github.com/vmware-tanzu/velero/pkg/builder" | grep -v "mocks" |
```

Comment:
I would simplify command and be more explicit about what to exclude

all output lines of command will containe `github.com/vmware-tanzu/velero/pkg/`, so would remove long string

```sh
go list ./pkg/... | grep -vE 'pkg/builder|pkg/apis|pkg/test|pkg/generated|pkg/plugin/generated|mocks'
```

Reply:
also, add task here https://github.com/vmware-tanzu/velero/issues/6190 to remove excludes not needed after code removal

Reply:
Sure. thanks